main.nf

```diff
@@ -1127,6 +1132,7 @@ if (!params.skipAlignment) {
                      -U $reads \\
                      $rnastrandness \\
                      --known-splicesite-infile $alignment_splicesites \\
+		       $hisat2_mp \\
```

Comment:
Minor thing - for optional arguments like this which will typically be empty for most people, I usually don't put them on their own line. Otherwise it looks odd when it's not specified and the command has a random empty line (though the `\` line break character should mean that it works fine).